source/deployment/guest-accounts.rst

```diff
@@ -3,63 +3,105 @@
 
 To invite guests into one or more Mattermost channels:
 
-1. Navigate to **Main Menu > Invite People**.  This is a new invitation option that combines inviting guests and members and consolidates **Send Email**, **Get Team Invite Link**, and **Add Members to the Team**. 
-2. Select **Invite Guests**.
-3. Enter the guest’s email address.
-4. Choose the channels the guest can join. 
-5. (Optional) Enter a custom message.
+1. Enable email invites from **System Console > Signup > Enable Email Invitations**.
+2. Navigate to **Main Menu > Invite People**.  This is a new invitation option that combines inviting guests and members and consolidates **Send Email**, **Get Team Invite Link**, and **Add Members to the Team**.
+3. Select **Invite Guests**.
+4. Enter the guest’s email address.
+5. Choose the channels the guest can join (excluding managed teams).
+6. (Optional) Enter a custom message.
 
 .. image:: ../images/Guest_Invite_Screen.png
 
-Guest Permission Settings 
+Configuring AD/LDAP Authentication
+----------------------------------
+
+When enabled, the **Guest Filter** in Mattermost identifies external users whose AD/LDAP role is guest and who are invited to join 
+your Mattermost server. These users will have the Guest role applied immediately upon first sign-in instead of the default member user role.
+This eliminates having to manually assign the role in the System Console.
+
+1. Enable Guest Access via **System Console > Guest Access (Beta)**.
+2. Navigate to **System Console > Authentication > AD/LDAP**.
+3. Complete the **Guest Filter** field.
+4. Choose **Save**.
+
+If a Mattermost Guest user has the guest role removed in the AD/LDAP system, the synchronization process
+will not automatically promote them to a member user role. This is done manually via **System Console > User Management**. If a
+member user has the **Guest Attribute** added, the synchronization processes will automatically demote the member user to the guest role.
+
+Configuring SAML 2.0 Authentication
+------------------------------------
+
+When enabled, the **Guest Attribute** in Mattermost identifies external users whose SAML assertion is guest and who are invited to join
+your Mattermost server. These users will have the Guest role applied immediately upon first sign-in instead of the default member user role.
+This eliminates having to manually assign the role in the System Console.
+
+If a Mattermost Guest user has the guest role removed in the SAML system, the synchronization processes will not automatically
+promote them to a member user role. This is done manually via **System Console > User Management**.
+If a member user has the **Guest Attribute** added, the synchronization processes will automatically demote the member user to the guest role.
+
+1. Enable Guest Access via **System Console > Guest Access (Beta)**.
+2. Navigate to **System Console > Authentication > SAML 2.0**.
+3. Complete the **Guest Attribute** field.
+4. Choose **Save**.
+
+
```

Comment:
Same as above, worth noting that a Guest will be taken to a generic screen letting them know they have no channels assigned until they have been added by a role with invite guest permissions.